nodeServerless/Local CE SAM/CE_lambda_functions/card_get.js

```diff
@@ -0,0 +1,62 @@
+/**
+ *
+ * Event doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
+ * @param {Object} event - API Gateway Lambda Proxy Input Format
+ *
+ * Context doc: https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-context.html 
+ * @param {Object} context
+ *
+ * Return doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html
+ * @returns {Object} object - API Gateway Lambda Proxy Output Format
+ * 
+ */
+
+let response;
+var AWS = require('aws-sdk');
+AWS.config.update({
+    region: "us-west-2"
+})
+
+let docClient = new AWS.DynamoDB.DocumentClient();
+// let ddb = new AWS.DynamoDB();
+
+exports.lambdaHandler = async (event, context) => {
+    try {
+        console.log("\n\n*********************\n");
+        var id = event.queryStringParameters.scryfallid;
+        var name = event.queryStringParameters.name;
+
+        console.log("id: " + id + ", type: " + typeof id + ". name: " + name + ", type: " + typeof name);
+        
+        var params = {
+            TableName: "BulkMagicCards",
```

Comment:
This table name is weird. Perhaps just MagicCards?

Reply:
Can't change a table name in dynamodb. I'm planning on rewriting my code for bulk uploading in node instead of java, will switch to a new table then.

Reply:
This should be trivial to do if you tables are defined in CloudFormation. That's fine to punt. Why java?

Reply:
I was planning on doing the project in Java but a wise mentor taught me that strongly typed is a pita so I'm learning node instead.
Not sure what you mean by "This should be trivial to do if you tables are defined in CloudFormation"

Reply:
Set it up like this: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dynamodb-table.html#aws-resource-dynamodb-table-syntax

Then you can just change the name and update your stack. You can even use the DDB table name  output as a Lambda environment variable to autowire your table name, but that may be a little too fancy ;)

Reply:
This is a cool way to do it. Looks like it only accepts string, number, boolean for data types. How should I deal with the lists in my json object?

Reply:
There is absolutely a list type in DDB. If you're in node, feel free to go schemaless.